roles/control_center_next_gen/defaults/main.yml

```diff
@@ -2,20 +2,26 @@
 # More control_center_next_gen variables are defined in the variables role
 
 ### Boolean to reconfigure Control Center Next Gen's logging with RollingFileAppender and log cleanup
-control_center_next_gen_custom_log4j: "{{ custom_log4j }}"
+control_center_next_gen_custom_log4j2: "{{ custom_log4j2 }}"
 
-### Root logger within Control Center Next Gen's log4j config. Only honored if control_center_next_gen_custom_log4j: true
-control_center_next_gen_log4j_root_logger: "INFO, main"
+### Root logger level within Control Center Next Gen's log4j2 config. Only honored if control_center_next_gen_custom_log4j2: true
+control_center_next_gen_log4j2_root_logger_level: "INFO"
 
-### Max number of log files generated by Control Center Next Gen. Only honored if control_center_next_gen_custom_log4j: true
+### Root logger appender within Control Center Next Gen's log4j2 config. Only honored if control_center_next_gen_custom_log4j2: true
+control_center_next_gen_log4j2_root_appenders:
+  - STDOUT
+  - ConnectAppender
```

Comment:
the values should correspond to the previous log4j2 appenders, should be "main" for control center. might also need to check the default log4j2.yaml to confirm the appenders